src/host/ut_host/ScreenBufferTests.cpp

```diff
@@ -1844,7 +1844,9 @@ void ScreenBufferTests::VtEraseAllPersistCursorFillColor()
         L"Change the colors to dark_red on bright_blue, then execute a Erase All.\n"
         L"The viewport should be full of dark_red on bright_blue"));
 
-    auto expectedAttr = TextAttribute(XtermToLegacy(1, 12));
+    auto expectedAttr = TextAttribute{};
+    expectedAttr.SetIndexedForeground((BYTE)XtermToWindowsIndex(1));
+    expectedAttr.SetIndexedBackground((BYTE)XtermToWindowsIndex(12));
     stateMachine.ProcessString(L"\x1b[31;104m");

```

Comment:
(It seems like this needed to change because of the treatment of legacy as 256=true in TextAttribute.hpp)

Reply:
Yep. But as I said, this was a deliberate choice. I didn't think it was appropriate to have legacy dark red map to ansi red, because I don't think it should be possible to brighten a legacy attribute.